src/helpers.coffee

```diff
@@ -244,6 +248,52 @@ exports.nameWhitespaceCharacter = (string) ->
     when '\t' then 'tab'
     else string
 
+exports.locationDataToAst = ({first_line, first_column, last_line, last_column, range}) ->
```

Comment:
@GeoffreyBooth thought about this a bit more and I prefer updating `Babylon` -> `Ast` in these names (rather than `Babel`). It unifies the AST-related naming and is a bit less "specific" than `Babel`, which could be confusing (for those who aren't already familiar with the codebase) eg "are we using Babel?"

So updated `locationDataToBabylon()` -> `locationDataToAst()`, `babylonLocationFields` -> `astLocationFields`

Reply:
But we're explicitly using the Babel AST _spec,_ it's linked to in a comment.